domain/src/main/java/com/teamagam/gimelgimel/domain/layers/LoadAllCachedLayersInteractor.java

```diff
@@ -36,6 +36,13 @@ public LoadAllCachedLayersInteractor(ThreadExecutor threadExecutor,
     private Observable<VectorLayer> createObservable() {
         return Observable.just(null)
                 .flatMapIterable(x -> mLayersLocalCache.getAllCachedLayers())
+                .map(this::recreateAsUnimportant)
                 .doOnNext(vl -> mProcessNewVectorLayerInteractorFactory.create(vl, null).execute());
     }
+
+    private VectorLayer recreateAsUnimportant(VectorLayer vectorLayer) {
```

Comment:
Maybe this should reside in VectorLayer as a static method. Unless you consider it a logic operation? It's like a dedicated builder